docs/mkdocs/docs/faq.md

```diff
@@ -40,24 +40,50 @@ of both batch and streaming data.
 - Support for "dynamic schemas" - ArcticDB supports datasets with changing schemas (column sets) over time.
 - Support for automatic data deduplication.
 
-###*What sort of data is ArcticDB best suited to?*
+### *What sort of data is ArcticDB best suited to?*
 
 ArcticDB is an OLA(nalytical)P DBMS, rather than an OLT(ransactional)P DBMS.
 
 In practice, this means that ArcticDB is optimised for large numerical datasets
 and for queries that operate over many rows at a time.
 
-###*Does ArcticDB require a server?*
+### *Does ArcticDB require a server?*
 
 No. ArcticDB is a fully fledged embedded analytical database system,
 designed for modern cloud and on-premises object storage that does not require
 a server for any of the core features.
 
-###*What languages can I use ArcticDB with?*
+### *What languages can I use ArcticDB with?*
 
 Bindings are currently only available for Python. 
 
-###*How can I get started using ArcticDB?*
+### *What is the best practice for saving Data to ArcticDB?*
+
+Users should consider how they store data based on their specific use cases. We have a guide [here](tutorials/data_organisation.md) that can help you determine what is best for you!
```

Comment:
More factual language, just `See the guide [here](tutorials/data_organisation.md)` is fine.